backend/src/core/services/job.service.ts

```diff
@@ -14,14 +14,40 @@ import { ListService } from '.'
 const createJob = async ({
   campaignId,
   rate,
+  scheduledTiming = undefined,
 }: {
   campaignId: number
   rate: number
+  scheduledTiming?: Date
 }): Promise<number | undefined> => {
+  if (!scheduledTiming) {
+    // if its not a scheduled campaign, just throw in current timing. will not affect main flow of logic
+    scheduledTiming = new Date()
+  }
+
+  //format scheduled Timing to a formatted string
+  // i think i needa offset 8 hrs man
```

Comment:
For comments that are for review, can u open a PR and then comment there instead of commenting directly in the code, so we can:
- Ensure that no such comment will get committed to the codebase
- Document our discussion on Github

Reply:
removed